src/js/proxyRequestFactory.js

```diff
@@ -0,0 +1,11 @@
+import apiFactory from './requestsFactory.js';
```

Comment:
can you implement proxy inside RequestsFactory, the separate object for these needs seems redundant